src/actions/config.js

```diff
@@ -82,6 +83,25 @@ export function configDidLoad(config) {
   };
 }
 
+const URL_BASE = 'https://raw.githubusercontent.com/netlify/netlify-cms/master/src/'
```

Comment:
@vencax i18n strings would need to be released in patch updates just like any other change.

For right now this can all go straight in the CMS source, I don't see it hurting us any time soon. Down the road we can convert languages into plugins, since a lot of things are going to become plugins, but no worries for now.